CareKit/CareKit.xcodeproj/project.pbxproj

```diff
@@ -1130,16 +1131,19 @@
 				MTL_FAST_MATH = YES;
 				PRODUCT_BUNDLE_IDENTIFIER = Apple.CareKitTests;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SUPPORTED_PLATFORMS = "watchsimulator watchos iphonesimulator iphoneos";
+				SUPPORTS_MACCATALYST = YES;
```

Comment:
When I set the project to build for iOS/watchOS, it automatically added catalyst, I don't see this directly in the Xcode IDE, but I suspect it doesn't hurt much